Server/NiKHost2.c

```diff
@@ -597,7 +598,7 @@ void textinfo(struct RexxMsg *mess) {
                                 ts->tm_mon + 1, ts->tm_mday, ts->tm_hour,
                                 ts->tm_min);
                         break;
-                defualt :
```

Comment:
Well, you can have labels, gotos, and other loop constructs in really weird ways in switch statements: https://en.wikipedia.org/wiki/Duff%27s_device
vbcc actually complained about the "defualt" label being unused.